provider/uprops/src/uprops_serde.rs

```diff
@@ -78,3 +78,28 @@ pub mod enumerated {
         pub enum_property: Vec<EnumeratedPropertyMap>,
     }
 }
+
+#[cfg(feature = "casemapping")]
+pub mod case {
+    #[derive(serde::Deserialize)]
+    pub struct Exceptions {
+        pub exceptions: Vec<u16>,
+    }
+
+    #[derive(serde::Deserialize)]
+    pub struct Unfold {
+        pub unfold: Vec<u16>,
+    }
+
+    #[derive(serde::Deserialize)]
+    pub struct Level1 {
+        pub code_point_trie: super::enumerated::EnumeratedPropertyCodePointTrie,
```

Comment:
Nit: Rename `EnumeratedPropertyCodePointTrie` to something more general.

Reply:
`uprops_serde::enumerated::EnumeratedPropertyCodePointTrie` -> `uprops_serde::SerializedCodePointTrie`